sdk/test/Services/AutoScaling/IntegrationTests/Config/462/App.config

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't know why, but in some places this diff seems to think a new file was added (when what happened instead was I renamed them from `45/App.config` to `462/App.config`, like this: https://github.com/aws/aws-sdk-net/pull/3202/files?diff=split&w=1#diff-8ddb50d4f9f3924b59e5f4a683dfc3c2c7b4bea75991dbaa3169a3cdcb88ee04).